src/pages/alternatives.js

```diff
@@ -19,6 +24,11 @@ export const query = graphql`
         contact_phone
       }
     }
+    alternativesImage: imageSharp(id: {regex: "/alternatives1/"}) {
+      sizes(maxWidth: 1240) {
```

Comment:
so `maxWidth` is just what we'd set for the largest version of the desktop design and then we don't have to do anything to have it auto size for other smaller resolutions?